source/administration/config-settings.rst

```diff
@@ -2042,9 +2042,22 @@ This filter uses the permissions of the **Bind Username** account to execute the
 | This feature's ``config.json`` setting is ``"UserFilter": ""`` with string input.                                                                                    |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Guest Filter
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+(Optional) Enter an AD/LDAP Filter to use when searching for external users who have Guest Access to Mattermost. Only the objects selected by the query will be accessible to Mattermost.
+This filter is used only when AD/LDAP Group Sync is enabled. See `AD/LDAP Group Sync documentation <https://docs.mattermost.com/deployment/ldap-group-sync.html>`__ for more information
```

Comment:
Omit this, this is not accurate.  The filter is used independently of the Group feature.  Potentially we can link the Guest accounts feature doc here so admins can learn more about the feature?

Reply:
Looks like that's my bad copy and paste job. So the second paragraph should be removed, or both? Happy to provide a link.

Reply:
Link added.